recipe/meta.yaml

```diff
@@ -106,6 +110,9 @@ outputs:
       #   - lib/python3.8/lib-dynload/_hashlib.cpython-38-x86_64-linux-gnu.so  # [linux]
       #   - lib/libpython3.dylib  # [osx]
       # match python.org compiler standard
+      skip: True            # [win and int(float(vc)) < 14]
+      skip_compile_pyc:
+        - '*.py'            # [osx and arm64]
```

Comment:
Is this going to be needed in all osx-arm64 packages?  Anything downstream package that uses python during the build will produce .pyc files in the standard library which will likely start clobbering each other.

Reply:
No. I manually compile them in the `build.sh` using the build python (same version). Otherwise `conda-build` will try to use the `host` python to compile them in the post build phase. `conda-build` has a check whether the `pyc` are already there and only compiles them if they are not. There are 2 issues with that though. One is that there are tests with python2 syntax that fails. Second is that conda-build check is wrong. For eg: for `foo/__init__.py` conda-build checks that `foo/__pycache__/__init__.pyc` is present. It should be `foo/__pycache__/__init__.cpython-38.pyc` though.

Reply:
Sounds good